src/query/pipeline/transforms/src/processors/transforms/transform_retry_async.rs

```diff
@@ -30,12 +30,16 @@ pub struct RetryStrategy {
 }
 
 pub struct AsyncRetryWrapper<T: AsyncRetry + 'static> {
+    retries: usize,
```

Comment:
better to use `Arc<AtomicUsize>`

Reply:
We want to let it be counted in each thread. So this variable is not shared between processors.